src/core/helpers.ts

```diff
@@ -23,3 +23,12 @@ export function generateID(): string {
 export function isEmptyObject(obj: Record<any, any>): boolean {
     return Object.keys(obj).length === 0;
 }
+
+export function findInArray<T>(array: T[], predicate: (entry: T) => boolean): T | undefined {
```

Comment:
Why don't use [Array.prototype.find()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find)?

Reply:
We're targeting es5 (as for now)